documentation/developer-guide/modules/tooling-guide/pages/samm-cli.adoc

```diff
@@ -193,6 +195,12 @@ resourceId:
   schema:
     type: string
 ----
+Definitions of the parameters must correspond to the specification of the OpenAPI _Parameter Object_.
+
+A template can be used for more specific customization - parameter `--template-file`.
+The template is a document in the OpenAPI format, with one crucial difference - the presence of a templated Paths Object `paths.$$__DEFAULT_QUERIES_TEMPLATE__$$`.
+All values not defined by the generator will be taken from the template.
+For generator-defined aspect model resources, the values can be taken from the corresponding _Operation Object_ template `paths.$$__DEFAULT_QUERIES_TEMPLATE__$$`.
```

Comment:
There really should be an example here to clarify how to use the DEFAULT_QUERIES_TEMPLATE

```suggestion
For generator-defined Aspect Model resources, the values can be taken from the corresponding _Operation Object_ template `paths.$$__DEFAULT_QUERIES_TEMPLATE__$$`.
An example for an OpenAPI generation template file to use with the `--template-file` argument is given below in YAML format:

`` `yaml
...
`` `
```